nextflow.config

```diff
@@ -59,7 +59,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base'
+    validationSchemaIgnoreParams     = ""
     validationShowHiddenParams       = false
```

Comment:
These can be removed! only `validate_params` should stay ;)

Reply:
Unless you want to keep the parameters of course

Reply:
the params are used to configurate the plugin
https://github.com/nf-cmgg/preprocessing/pull/97/files#diff-ce7465a8e67b3b9c95696db1146ca7e6328f075e08a3d64062b87aa8608fc4eaR252-R265

Reply:
Yes I know but what is the benefit of doing this?